_posts/2019-11-27-making-your-clients-better-product-owners.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "Making Your Clients Better Product Owners"
+date: 2019-11-27 12:30:00
+categories: ["Scrum", "Agile"]
+author: "abizzinotto"
+---
+
+Different clients bring different projects, perspectives, workflows and experiences, as well as different challenges. Before starting a project, one of those challenges is to define who will be the Product Owner. 
+
+Ideally, you would be able to assign the role internally or to the client based solely on the characteristics of the project. However, that's not always the case. It might just be that the client insists on being the Product Owner or that you are a small team and can't really assign the role internally. Whatever the reason, you might end up in a situation where your client isn't really a good Product Owner.
+
+Here I'll share some strategies we implement to help our clients become better Product Owners and ensure the best experience for them and for our team.
+
+<!--more-->
+
+##Communicate expectations clearly
+
+Not every client is familiar with the responsibilities of the Product Owner. Not every client is even familiar with Scrum as a whole. Therefore, it is important to communicate expectations clearly at the beginning of the project.
```

Comment:
@ABizzinotto Maybe you can add a link when you mention `Scrum`

Reply:
@lubc do you mean a link like this? https://www.scrumguides.org/scrum-guide.html#definition

Reply:
@lubc actually, we ended up replacing 'Scrum' :)